lib/color_spinor_util.cu

```diff
@@ -434,10 +433,9 @@ namespace quda {
     for (int s = 0; s < Ns; s++) {
       printfQuda("x = %u, s = %d, { ", i, s);
```

Comment:
In light of the below changes, should this also be a vanilla `printf`? Also, do we want to do one "big" `sprintf` build then `printf` so we can guarantee everything for a single coordinate/color/spin gets printed together (for multi-rank cases)?

Reply:
Also, any good argument for using `printf` here and `std::out` above ?

Reply:
Unless things have changed, `printfQuda()` should be used for all output, which ensures that the "prefix" and "outfile" options of `setVerbosityQuda()` are respected.

Reply:
@weinbe2 Good catch, thank you.

@rbabich The motivation here was that `printfQuda` always only prints from process 0.  In this specific case we want to printing from all processes since it's a specific debugging routine.  However, looking at the source code now, I see that it is possible to achieve what we want with `getRankVerbosity`.  I have no memory of adding that 😄 , but it appears that I did a few years ago - with this in mind I'll change this routine back to using `printfQuda` leaving the runtime opt-in if multiple processes with the `QUDA_RANK_VERBOSITY` env.